layouts/index.html

```diff
@@ -27,11 +27,11 @@ <h1 class="usa-display">{{ $translation.homepage.header }}</h1>
             </option>
             {{ if eq $.Site.Params.language "english" }}
               {{ range where .Data.Pages.ByTitle "Section" "register" }}
-                <option value="{{ replace .Title "." "" | lower | urlize }}">{{ title .Title }}</option>
+                <option value="{{ .Params.state_abbreviation | lower }}">{{ title .Title }}</option>
```

Comment:
Oh, wait - if we do that, then it'll always pick the English `.Title`, won't it?